projects/igniteui-angular/src/lib/combo/combo-dropdown.component.ts

```diff
@@ -322,7 +320,7 @@ export class IgxComboDropDownComponent extends IgxDropDownBase {
     }
 
     onToggleClosing() {
+        this.verticalScrollContainer.scrollTo(0);
```

Comment:
I thought we said that the scroll position will be reset on searching, instead of always?

Reply:
@kdinev read my last comment in the issue and please share your thoughts. Thanks!

Reply:
I thought what we said yesterday is that the scrolling is preserved to the last scroll position, unless searching is initiated?